docs/encyclopedia/workers/worker-deployments.mdx

```diff
@@ -0,0 +1,85 @@
+
+Worker Deployments is available in [Pre-release](/evaluate/development-production-features/release-stages#pre-release).
+
+Clients:
+
+- Temporal CLI: version [v1.3.0](https://github.com/temporalio/cli/releases/tag/v1.3.0) or higher
+- Go SDK: version [v1.33.0](https://github.com/temporalio/sdk-go/releases/tag/v1.33.0) or higher
+- Other SDKs: coming soon!
+
+Open source users need:
+
+- Temporal Server: version [v1.27.1](https://github.com/temporalio/temporal/releases/tag/v1.27.1) or higher
+- Open Source UI: version [v2.36.0](https://github.com/temporalio/ui/releases/tag/v2.36.0) or higher
+
+:::
+
+## What are Worker Deployments? {#deployments}
+
+A Worker Deployment is a logical service that groups similar Workers together for unified management.
+
+Each Deployment has a name (such as your service name) and supports versioning through a series of [Worker Deployment Versions](#deployment-versions).
+
+## What are Worker Deployment Versions? {#deployment-versions}
+
+A Worker Deployment Version represents an iteration of a Worker Deployment.
+
+Each Deployment Version consists of Workers that share the same code build and environment.
+
+When a Worker starts polling for Workflow and Activity Tasks, it reports its Deployment Version to the Temporal Server.
+
+## What can I do with Worker Deployments? {#use-cases}
+
+Worker Deployments currently support [Worker Versioning](/worker-versioning), with more features planned for the future.
+
+## How do I enable Worker Deployments? {#how-to}
+
+:::note
+
+Temporal Server disables Worker Deployments by default.
+
+Temporal Cloud customers: Contact your Temporal account team or the [Temporal Support Team](https://docs.temporal.io/cloud/support#support-ticket) for Pre-release access.
+
+Open source users: Enable the [system.enableDeployments](https://github.com/temporalio/temporal/blob/main/common/dynamicconfig/constants.go) and [frontend.workerVersioningWorkflowAPIs](https://github.com/temporalio/temporal/blob/main/common/dynamicconfig/constants.go) dynamic config values by setting both to `true`.
+
+For example, with the Temporal CLI, run:
+
+```command
+temporal server start-dev \
+   --dynamic-config-value system.enableDeployments=true \
+   --dynamic-config-value frontend.workerVersioningWorkflowAPIs=true
+```
+
+:::
+
+To add your Worker to a Deployment, provide both the Deployment name and Version when creating your Worker.
+
+When running multiple Workers in one process (such as when using multiple [Task Queues](/task-queue)), you'll typically use the same Deployment name for all of them since they belong to the same service.
+
+The server automatically registers new Deployments and Deployment Versions when a Worker connects.
+
+Per-SDK documentation will be available soon.
+
+This change is safe as long as you don't enable Versioning on that deployment. See the [Worker Versioning](/worker-versioning) docs for information on preparing your deployment system.
```

Comment:
I don't think we should encourage users passing this info on unversioned workers just yet, because (1) there is no benefit to that. (2) until the design is final for rolling deployments we don't know if the statement of "This change is safe as long as you don't enable Versioning" is 100% true.